scripts/recipes.update/update_modui.sh

```diff
@@ -9,13 +9,14 @@ if [[ "$changed" -eq 1 ]]; then
 fi
 
 cd $ZYNTHIAN_SW_DIR/mod-ui
-git pull | grep -q -v 'Already up.to.date.' && changed=1
-if [[ "$changed" -eq 1 ]]; then
-	#pip3 install -r requirements.txt
-	cd utils
-	make -j 4
-	cd ..
-fi
+git remote remove origin
+git remote add origin https://github.com/zynthian/mod-ui.git
+git fetch origin zyn-mod-merge
+git checkout zyn-mod-merge
+git reset --hard origin/zyn-mod-merge
+
+cd utils
+make -j 4
```

Comment:
Hi @steveb !
This script is run every time zynthian is updated, so it should detect the current "origin" and run the origin-change code when it's needed only. Also, when new changes are pulled, it should run make.

Reply:
FWIW, there is no time penalty for running make when there are no changes, nor is there a time penalty for always resetting the remote. The only time taken is doing the pull/fetch from github:

```
# time bash update_modui.sh 
From https://github.com/zynthian/mod-ui
 * branch              zyn-mod-merge -> FETCH_HEAD
 * [new branch]        zyn-mod-merge -> origin/zyn-mod-merge
Already on 'zyn-mod-merge'
HEAD is now at b8216b9c [zynthian] Add ZynMidiRouter ports to MOD-UI: main_in & main_out
make: Nothing to be done for 'all'.

real	0m5.836s
user	0m0.884s
sys	0m0.478s
```

Reply:
OK! Good point ... ;-)